src/ManageCourses.Domain/DatabaseAccess/ManageCoursesDbContext.cs

```diff
@@ -339,6 +339,16 @@ public IQueryable<McUser> GetMcUsers(string email)
             return users;
         }
 
+        public UcasInstitution GetUcasInstitution(string name, string instCode)
+        {
+            return UcasInstitutions.FromSql(@"
+                SELECT i.* from ucas_institution i
+                JOIN mc_organisation_institution oi on i.inst_code = oi.institution_code
+                JOIN mc_organisation_user ou on oi.org_id = ou.org_id
+                WHERE lower(ou.email) = lower(@email)",
+                new NpgsqlParameter("email", name)).FirstOrDefault();
+        }
```

Comment:
It may be worth considering some includes here?

Reply:
Good point! I included UcasCampuses at this point. :+1: